tools/watchexec_script.sh

```diff
@@ -0,0 +1,3 @@
+# SPDX-License-Identifier: MPL-2.0
+
+watchexec -w $1 -r -c -- "make build && make format && make check"
```

Comment:
why not use `watch` instead? `watch` is present on most ubuntu/debian distros by default

Reply:
watchexec waits for the program date to be updated.